src/Cardano/TextEnvelope.purs

```diff
@@ -0,0 +1,100 @@
+module Cardano.TextEnvelope
```

Comment:
Please re-export under `Contract.TextEnvelope` to make it clear this is a stable API.